### ISSUE TITLE
[Support]: Won't start, [Errno 113] No route to host, after mqtt address change

### ISSUE THREAD

**Issue by Perkinje** [blakeblackshear/frigate#3989]
### Describe the problem you are having

Seemed to be running fine. Added a new camera to the config noticed my mqtt ip address was wrong. Previous mqtt server running as an addon to Home Assistant (HA). HA had a hardware failure and when I recreated it, I switched to using an mqtt docker in Truenas Scale. I've had zero issues from HA using that mqtt server and can connect to the server using mqtt explorer. I forgot to change the mqtt address in Frigate, but, oddly, it was working fine through the HA integration with the old mqtt address. After the modifying the mqtt address and adding the additional camera, I restarted the docker and it failed to load completely with the no route to host error. I removed the extra camera config and retried with the same outcome.

I found another post with a similar issue but on a QNAP setup (issue #1272). They solved the problem with a short script delaying the migration attempt.
```
#!/usr/bin/bash
set -x
echo "Sleep for Frigate zzz"
sleep 20
python3 -u -m frigate
sleep 20
echo "try again..."
python3 -u -m frigate
```
I tried this and the script was executed appropriately but I had the same outcome as before. Not sure what else to try at this point.

### Version

Not sure since I can't get it started (Docker image blakeblackshear/frigate:stable-amd64)

### Frigate config file

```yaml
mqtt:
  host: 192.168.1.158
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  stats_interval: 60

detectors:
  coral:
    type: edgetpu
    device: usb

database:
  path: /db/frigate.db

logger:
  default: debug

ffmpeg:
  global_args: -hide_banner -loglevel debug
  hwaccel_args:
     - -hwaccel
     - vaapi
     - -hwaccel_device
     - amd
     - /dev/dri/renderD128
     - -hwaccel_output_format
     - yuv420p

objects:
  track:
    - person
    - dog
    - package
  filters:
    person:
      min_area: 5000
      max_area: 100000
      min_score: 0.6
      threshold: 0.8
    dog:
      min_area: 5000
      max_area: 100000
      min_score: 0.6
      threshold: 0.79

record:
  enabled: True
  retain:
    days: 2
    mode: all
  events:
    retain:
      default: 10
    pre_capture: 20
    post_capture: 30
    objects:
      - person
      - dog
      - package
    retain:
     default: 12
     mode: motion
     objects:
       person: 15

detect:
  fps: 7
  max_disappeared: 30
  stationary:
    interval: 0
    threshold: 50

live:
  height: 720
  quality: 5

rtmp:
  enabled: False

cameras:
  Doorbell:
    ffmpeg:
      inputs:
        - path: 'rtsp://ubnt.*************@192.168.1.1:7447/*************'
          roles:
            - detect
    rtmp:
      enabled: False
    detect:
      width: 1600
      height: 1200

  Living_Room:
    ffmpeg:
      inputs:
        - path: 'rtsp://ubnt.*************@192.168.1.1:7447/*************'
          roles:
            - detect
    rtmp:
      enabled: False
    detect:
      width: 1920
      height: 1080
    motion:
      threshold: 25
      mask:
        - 1211,0,1211,280,696,271,681,26

  Office:
    ffmpeg:
      inputs:
        - path: 'rtsp://ubnt.*************@192.168.1.1:7447/*************'
          roles:
            - detect
    rtmp:
      enabled: False
    detect:
      width: 1920
      height: 1080

  Driveway:
    ffmpeg:
      inputs:
        - path: 'rtsp://ubnt.*************@192.168.1.1:7447/*************'
          roles:
            - detect
    rtmp:
      enabled: False
    record:
      events:
        required_zones:
          - street
          - drive
    snapshots:
      required_zones:
        - street
        - drive
    detect:
      width: 1920
      height: 1080
    zones:
      drive:
        coordinates: 1729,1080,1316,168,401,179,120,484,0,698,0,939,0,1080
        objects:
          - person
          - car
      street:
        coordinates: 1305,141,1663,195,1648,130,1614,107,1506,85,1095,76,899,65,760,71,566,89,274,121,35,163,23,231,554,150,892,143
        objects:
         - person

  Backyard_Upper_Deck:
    ffmpeg:
      inputs:
        - path: 'rtsp://ubnt.*************@192.168.1.1:7447/*************'
          roles:
            - detect
    record:
      events:
        required_zones:
          - hot_tub
    snapshots:
      required_zones:
        - hot_tub
    rtmp:
      enabled: False
    detect:
      width: 1920
      height: 1080
    zones:
      hot_tub:
        coordinates: 722,897,745,758,731,752,582,775,519,796,466,931
        objects:
          - person
          - dog
```

### Relevant log output

```shell
frigate  | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
frigate  | [s6-init] ensuring user provided files have correct perms...exited 0.
frigate  | [fix-attrs.d] applying ownership & permissions fixes...
frigate  | [fix-attrs.d] done.
frigate  | [cont-init.d] executing container initialization scripts...
frigate  | [cont-init.d] done.
frigate  | [services.d] starting services
frigate  | [services.d] done.
frigate  | [2022-09-30 12:39:42] frigate.app                    INFO    : Starting Frigate (0.10.1-83481af)
frigate  | Starting migrations
frigate  | [2022-09-30 12:39:42] peewee_migrate                 INFO    : Starting migrations
frigate  | There is nothing to migrate
frigate  | [2022-09-30 12:39:42] peewee_migrate                 INFO    : There is nothing to migrate
frigate  | [Errno 113] No route to host
frigate  | [cmd] python3 exited 1
frigate  | [cont-finish.d] executing container finish scripts...
frigate  | [cont-finish.d] done.
frigate  | [s6-finish] waiting for services.
frigate  | [s6-finish] sending all processes the TERM signal.
frigate  | [s6-finish] sending all processes the KILL signal and exiting.
```

### FFprobe output from your camera

```shell
Nothing
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Unifi G3/G4 Pro, Flex and Instant

**Comment by NickM-27** [comment]
Is frigate running in the same IP subnet as the MQTT server? (192.168.1)

Seems like it might be some firewall or other issue keeping it from talking to it, since otherwise it shouldn't be an issue.

**Comment by Perkinje** [comment]
Yes and using Telnet I can connect to the mqtt port from the Frigate machine

**Comment by Perkinje** [comment]
Never-mind, I figured it out and I'm an idiot sometimes. I've wasted your time. I was modifying a backup file that I forgot to label as a .bak. In my compose file I reference a different config.yml file that I moved to a more central location with other files I may modify remotely. I've made the changes to the referenced config file and all is well now. 

Again, very sorry. I fell stupid at the moment as I've wasted a few hours of my time and yours. Just don't tell my wife...
